src/main/resources/templates/home.html

```diff
@@ -0,0 +1,68 @@
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="dropdown">
+            <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenu2" data-bs-toggle="dropdown" aria-expanded="false">
+                Menu
+            </button>
+            <ul class="dropdown-menu" aria-labelledby="dropdownMenu2">
+                <li><a class="dropdown-item" th:href="@{/}">Strona główna</a></li>
+                <li><a class="dropdown-item" th:href="@{/add}">Dodaj wpis</a></li>
+                <li><a class="dropdown-item" th:href="@{/filter?type=INCOME}">Wyświetl przychody</a></li>
+                <li><a class="dropdown-item" th:href="@{/filter?type=EXPENSE}">Wyświetl wydatki</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div th:unless="${transactions.isEmpty()}" class="container">
+    <div class="row">
+        <table class="table">
+            <thead>
+            <tr>
+                <th scope="col">#</th>
+                <th scope="col">Opis</th>
+                <th scope="col">Rodzaj wydatku</th>
+                <th scope="col">Kwota (zł)</th>
+                <th scope="col">Data</th>
+                <th scope="col">Opcje</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr th:each="transaction: ${transactions}">
+                <th scope="row" th:text="${transaction.getId()}"></th>
+                <td th:text="${transaction.getDescription()}"></td>
+                <td th:text="${transaction.getType().getDescription()}"></td>
+                <td th:text="${transaction.getAmount()}"></td>
+                <td th:text="${transaction.getDate()}"></td>
+                <td><a th:href="@{/edit(id=${transaction.getId()})}"><i class="bi bi-pencil-square"></i></a>
+                    <form th:action="@{/remove}" method="post" style="display:inline;">
+                        <input type="hidden" name="id" th:value="${transaction.getId()}">
+                        <button type="submit" class="btn btn-outline-danger">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-trash" viewBox="0 0 16 16">
+                                <path d="M5.5 5.5A.5.5 0 0 1 6 6v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5m2.5 0a.5.5 0 0 1 .5.5v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5m3 .5a.5.5 0 0 0-1 0v6a.5.5 0 0 0 1 0z"/>
+                                <path d="M14.5 3a1 1 0 0 1-1 1H13v9a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V4h-.5a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1H6a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1h3.5a1 1 0 0 1 1 1zM4.118 4 4 4.059V13a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V4.059L11.882 4zM2.5 3h11V2h-11z"/>
+                            </svg>
+                        </button>
+                    </form>
+                    </td>
```

Comment:
Przyciski edycji i usunięcia mają inny styl, przez co wygląda to niechlujnie 
<img width="117" alt="image" src="https://github.com/AdamJaniczek/home-budget/assets/2963232/6c19a526-1f34-4e63-a5ab-4c8474013203">